### ISSUE TITLE
[All Browsers] Validation Component: Spinbox's are not aligned correctly

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise#547]
**Describe the bug**
[All Browsers] Validation Component: Spinbox's are not aligned correctly

**To Reproduce**
1. Go to URL: http://490-rc1-sohoxi.demo.design.infor.com/components/validation/example-short-fields.html
2. Scroll down to Spinbox section and tab to each spinbox without clicking on the "+" or "-" button to get validation error
4. The spinbox's are misaligned 

**Expected behavior**
All the Spinbox's should be aligned

**Screenshots**
Please see screen shot attached. 
![validation_shortfield](https://user-images.githubusercontent.com/39101346/43592184-fdc30a3a-9642-11e8-99ff-dd74335c04b7.png)

**Platform**
 - All Browsers 

**Additional context**
None

**Comment by davidcarlsonberg** [comment]
This looks like intended design to me. What should they look/align like?

**Comment by EdwardCoyle** [comment]
So these are "short-field" spinbox components that are used in large, form-heavy pages (usually in legacy Infor apps).  The [regular-sized spinbox](http://490-rc1-sohoxi.demo.design.infor.com/components/spinbox/example-index.html) gets displayed like this:

<img width="199" alt="screen shot 2018-08-03 at 1 58 07 pm" src="https://user-images.githubusercontent.com/3249601/43657954-4af6c76e-9725-11e8-82e2-da0e734dabda.png">

Basically we want to shrink input fields that are inside `.short-field` wrapper elements down to match the size of the buttons here.

**Comment by tmcconechy** [comment]
Yeah, the middle input shouldn't "pop out" it should be connected on top and bottom.
Probably just needs a height set for the short-field class in spinbox.scss

**Comment by brianjuan** [comment]
@CindyMercadoReyes 
Middle input box in iOS devices is lightly smaller, is this fine?
<img width="530" alt="screen shot 2018-08-17 at 7 05 23 pm" src="https://user-images.githubusercontent.com/39508864/44263243-c02c1d00-a250-11e8-9c5d-0cf690f8a4d4.png">

**Comment by tmcconechy** [comment]
We should probably fix this @davidcarlsonberg , if you dont mind. We could add a rule on `ios`
Suggest to move this to the failed QA column for adjustment

**Comment by davidcarlsonberg** [comment]
Is there a chosen method for testing mobile devices in their native environment that developers can access? Browser dev tools are ineffective in this scenario.

**Comment by CindyMercadoReyes** [comment]
Team, the alignment issue for the spinbox's is now resolved. However, for dropdown the selected value is not appearing for "Custom".  In addition, on Win 10 IE 11 the error validation is not working. 
Please see screen shot attached. 

![bs_win10_ie_11 0 2](https://user-images.githubusercontent.com/39101346/44670152-ea6ba080-a9ef-11e8-91aa-9bf2c116cf13.jpg)

**Comment by tmcconechy** [comment]
What do you mean by 1 exactly @CindyMercadoReyes ? Im not following that part.
Do you mean the custom dropdown field is so short it cant show the value?

If so i see what you mean. I think these probably should be a new issue as its unrelated to this styling issue.

**Comment by tmcconechy** [comment]
We created a new issue for this problem. https://github.com/infor-design/enterprise/issues/673 so this should just be verified for QA next round.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved. Moving this ticket to the "Done" folder. Thanks, Cindy
